product_configurator_stock/views/stock_move_view.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
This line is no longer needed. (Not a blocker, just cosmetic)

Reply:
@patrickrwilson This encoding is correct for xml. The removal is only for python files with python >=3